run-codemods.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/zsh
+#! /bin/sh
 
 # Designed for use on Macs. May work on other *nix OSes.
```

Comment:
If we verify that things work on Macs and other *nix OSes by using sh instead of zsh, and using perl to determine the path of $0, this comment should either be removed, or updated to reflect where it has been tested.

Reply:
I'll adjust the shebang after deciding between js and sh

Reply:
I now see the comment you're referring to... I think it didn't show up until I opened the file changes and I just saw the comment attached to the shebang